### ISSUE TITLE
Sharing custom vehicle model no longer shares sensor configs at the same time

### ISSUE THREAD

**Issue by HaoruXue** [lgsvl/simulator#1976]
The WISE web interface seems to have had an update recently. Before when I share custom vehicles, the other people can also see my sensor configurations, while it is no longer true. Sharing custom vehicle no longer shares sensor config. 

An example below where the people being shared with a custom vehicle is not able to see the author's sensor config, but only the user's own sensor configs.

![image](https://user-images.githubusercontent.com/32390737/151914398-7609d263-6148-4726-aab4-c1c674af3d0f.png)

Is this an intended change?

**Comment by momala** [comment]
> The WISE web interface seems to have had an update recently. Before when I share custom vehicles, the other people can also see my sensor configurations, while it is no longer true. Sharing custom vehicle no longer shares sensor config.
> 
> An example below where the people being shared with a custom vehicle is not able to see the author's sensor config, but only the user's own sensor configs.
> 
> ![image](https://user-images.githubusercontent.com/32390737/151914398-7609d263-6148-4726-aab4-c1c674af3d0f.png)
> 
> Is this an intended change?

You should share your sensor's configuration separately [go to the sensor conf. page ]. previously it wasn't like that but now for some reason, they made it.

**Comment by EricBoiseLGSVL** [comment]
yes, the id for the sensor config also shares the vehicle it is made with.  It simplified api loading.
